README.md

```diff
@@ -1,78 +1,63 @@
+Description
+-----------
 
-## Description
-
-SOPT is a C++ package to perform Sparse OPTimisation. It solves a variety of sparse regularisation
-problems, including the SARA algorithm. Prototype Matlab implementations of various algorithms are
+**SOPT** is an opensource `C++` package to perform Sparse OPTimisation. It solves a variety of sparse regularisation
+problems, including the Sparsity Averaging Reweighted Analysis (SARA) algorithm. Prototype Matlab implementations of various algorithms are
 also included.
 
-## Contributors
-
-SOPT was initially created by Rafael Carrillo, Jason McEwen and Yves Wiaux but major contributions
-have since been made by a number of others. The full list of contributors is as follows:
-
-* [Rafael E. Carrillo](https://www.researchgate.net/profile/Rafael_Carrillo2)
-* [Jason D. McEwen](http://www.jasonmcewen.org)
-* [Yves Wiaux](http://basp.eps.hw.ac.uk)
-* [Vijay Kartik](https://people.epfl.ch/vijay.kartik)
-* [Mayeul d'Avezac](https://github.com/mdavezac)
-* [Luke Pratley](https://about.me/luke.pratley)
-* [David Perez-Suarez](https://dpshelio.github.io)
-
-## References
-When referencing this code, please cite our related papers:
-
-1. R. E. Carrillo, J. D. McEwen and Y. Wiaux. "Sparsity Averaging Reweighted
-   Analysis (SARA): a novel algorithm for radio-interferometric imaging", Mon.
-   Not. Roy. Astron. Soc., 426(2):1223-1234, 2012,
-   [arXiv:1205.3123](http://arxiv.org/abs/arXiv:1205.3123)
-1. R. E. Carrillo, J. D. McEwen, D. Van De Ville, J.-P. Thiran, and Y. Wiaux.  "Sparsity averaging
-   for compressive imaging", IEEE Signal Processing Letters, 20(6):591-594, 2013,
-   [arXiv:1208.2330](http://arxiv.org/abs/arXiv:1208.2330)
-1. A. Onose, R. E. Carrillo, A. Repetti, J. D. McEwen, J.-P. Thiran, J.-C. Pesquet, and Y. Wiaux.
-   "Scalable splitting algorithms for big-data interferometric imaging in the SKA era". Mon. Not.
-   Roy. Astron. Soc., 462(4):4314-4335, 2016,
-   [arXiv:1601.04026](http://arxiv.org/abs/arXiv:1601.04026)
-
-## Webpage
+Dependencies installation
+-------------------------
 
-http://basp-group.github.io/sopt/
+**SOPT** is mostly written both in `C++11` and `Python`. Pre-requisites and dependencies are listed in following and minimal versions required are tested against `Travis CI` meaning that they come natively with the Ubuntu Trusty release. These are also the default ones fetched by `CMake`.
 
-## Installation
+`C++` minimal dependencies:
 
-### C++ pre-requisites and dependencies
-
-- [CMake](http://www.cmake.org/): a free software that allows cross-platform compilation
-- [tiff](http://www.libtiff.org/): Tag Image File Format library
-- [OpenMP](http://openmp.org/wp/): Optional. Speeds up some of the operations.
-- [UCL/GreatCMakeCookOff](https://github.com/UCL/GreatCMakeCookOff): Collection of cmake recipes.
+- [CMake](http://www.cmake.org/) v3.9.2 A free software that allows cross-platform compilation
+- [GCC](https://gcc.gnu.org) v7.3.0 GNU compiler for `C++`
```

Comment:
This is the version needed for purify. I expect that sopt will work with much less.

Reply:
That's what is implemented in Travis. But I guess you won't use a different compiler version for purify and sopt ? That would be vicious.